client/gatsby-browser.js

```diff
@@ -10,6 +10,8 @@ import {
   DefaultLayout,
   GuideLayout
 } from './src/components/layouts';
+import GuideNavMenu from './src/components/layouts/components/guide/NavMenu';
+import DefaultNavMenu from './src/components/Header/components/NavMenu';
```

Comment:
@ojeytonwilliams I do not see how it is better. We have a default menu, and sometimes we replace it with a custom one. Why should we pass the default menu to the header all the time?